diff --git a/doc/doc-support/lib-function-docs.nix b/doc/doc-support/lib-function-docs.nix
index f6d613ca..244d029a 100644
--- a/doc/doc-support/lib-function-docs.nix
+++ b/doc/doc-support/lib-function-docs.nix
@@ -23,5 +23,10 @@ with pkgs; stdenv.mkDerivation {
     docgen debug 'Debugging functions'
     docgen options 'NixOS / nixpkgs option handling'
     docgen sources 'Source filtering functions'
+    docgen generators 'Generator functions'
+    docgen customisation 'Customisation functions'
+    docgen meta 'Meta functions'
+    docgen modules 'Modules functions'
+    docgen versions 'Versions functions'
   '';
 }
